src/pyenphase/updaters/meters.py

```diff
@@ -215,3 +218,26 @@ async def update(self, envoy_data: EnvoyData) -> None:
 
                 if phase_consumption:
                     envoy_data.ctmeter_consumption_phases = phase_consumption
+
+            # match meter identifier to storage meter found during probe
+            if meter["eid"] == self.storage_meter_eid and self.storage_meter_type:
+                # if storage meter was enabled (type known) store ctmeter storage data
+                envoy_data.ctmeter_storage = EnvoyMeterData.from_api(
+                    meter, meters_status[index]
+                )
+                # if more then 1 phase configured store ctmeter phase data
+                phase_storage: dict[str, EnvoyMeterData] = {
+                    PHASENAMES[phase_idx]: storage
+                    for phase_idx in range(
+                        self.phase_count if self.phase_count > 1 else 0
```

Comment:
This line appears a lot in this function now. Might make sense to make it a local var

Reply:
Changed